app/models/foreman_openscap/arf_report.rb

```diff
@@ -173,6 +173,17 @@ def destroy
       end
     end
 
+    def self.inner_select(taxonomy, inner_method = which_ancestry_method)
```

Comment:
please add a comment "# can be removed after http://projects.theforeman.org/issues/14458 is fixed"